README.md

```diff
@@ -10,15 +10,18 @@ The contributing process documentation is still in-progress. The numbered list b
 
 After you've created a fork of this repository under your GitHub account, and cloned your fork to your machine, the contribution flow looks like:
 
+1. Pick a [repo issue](https://github.com/React-Indy/website/issues) to work on.
 1. Create a new branch from `main`
-2. Make changes and commit them to your branch
-3. Create a PR with your branch against this repo's `main`
-4. Request review(s) of your PR
-5. Allow at least 24-48 hours for review
-6. Address any review comments
-7. Repeat steps 4-6 as needed
-8. Allow your PR to be approved & merged
-9. Celebrate! 🎉
+1. Make changes and commit them to your branch
```

Comment:
@pirate-matt do you have a MD formatter that is getting rid of the numbering of these lists? I feel like we probably want to keep the sequential numbering for DX

Reply:
Plus one to this -- can we update these numbers to be explicit?

Reply:
👍🏻 updated!